api/app/models/land_change/tree_cover_loss.py

```diff
@@ -0,0 +1,112 @@
+import json
+import uuid
+from typing import Union, List, Annotated, Literal, Optional
+from pydantic import Field, field_validator, model_validator
+from app.models.common.analysis import AnalysisStatus
+from app.models.common.base import StrictBaseModel, Response
+from app.models.common.areas_of_interest import (
+    AdminAreaOfInterest,
+    ProtectedAreaOfInterest,
+)
+
+
+AoiUnion = Union[
+    AdminAreaOfInterest,
+    ProtectedAreaOfInterest,
+]
+
+DATE_REGEX = r"^\d{4}$"
+AllowedCanopyCover = Literal[10, 15, 20, 25, 30, 50, 75]
+AllowedIntersections = List[Literal["driver", "natural_lands"]]
+
+
+class TreeCoverLossAnalyticsIn(StrictBaseModel):
+    aois: List[Annotated[AoiUnion, Field(discriminator="type")]] = Field(
+        ..., min_length=1, max_length=1, description="List of areas of interest."
+    )
+    start_year: str = Field(
+        ...,
+        title="Start Date",
+        description="Must be year in YYYY date format. Minimum year is 2001.",
+        pattern=DATE_REGEX,
+        examples=["2020", "2024"],
+    )
+    end_year: str = Field(
+        ...,
+        title="End Date",
+        description="Must be year in YYYY date format. Minimum year is 2001.",
+        pattern=DATE_REGEX,
+        examples=["2023", "2024"],
+    )
+    canopy_cover: AllowedCanopyCover = Field(
+        ...,
+        title="Canopy Cover",
+    )
+    intersections: AllowedIntersections = Field(
```

Comment:
fyi in this case, it was decided intersections are just primary forest and intact forests

Reply:
something like `forest_filter`

Reply:
Updated in 35df0949